Makefile.am

```diff
@@ -248,6 +249,17 @@ cpp_tools_ct_clustertool_SOURCES = \
 	cpp/util/util.cc \
 	cpp/version.cc
 
+cpp_tools_db_tool_LDADD = \
+	cpp/libcore.a \
+	$(libevent_LIBS) \
+	-lprotobuf -lsqlite3 -lleveldb
```

Comment:
Ah, I see the problem. You should be using `$(leveldb_LIBS)` rather than `-lleveldb`.